dns_cache/manager.go

```diff
@@ -0,0 +1,62 @@
+package dns_cache
```

Comment:
this package naming encourages to use snake case in the client code and considered to be not the best, more info here: https://golang.org/doc/effective_go.html#package-names